tools/setup/emoji/generate_emoji_names_table

```diff
@@ -139,11 +145,43 @@ def generate_emoji_code_to_emoji_names_maps() -> None:
             reverse_unified_reactions_map[emoji_code]
         )
 
-    # Prepare iamcal names map.
+    not_found_count = 0
+    print(f"\npossible CLDR emoji are: \n" + "".join(CLDR_DATA.keys()) + "\n\n-----\n")
+
+    # Prepare iamcal and CLDR names map.
     for emoji_dict in EMOJI_DATA:
         emoji_code = get_emoji_code(emoji_dict)
         emoji_code_to_iamcal_names[emoji_code] = ", ".join(emoji_dict["short_names"])
 
+        # create the unicode character(s) for the emoji, since this is the key into the CLDR data
+        emoji =  "".join([chr(int(h, 16)) for h in  emoji_dict['unified'].split('-')])
```

Comment:
emoji-datasource-google lists some emoji with a U+FE0F emoji-style variation selector. That explains 299 of your missing rows.

```suggestion
        emoji = "".join(
            chr(int(h, 16))
            for h in (emoji_dict["non_qualified"] or emoji_dict["unified"]).split("-")
        )
```

Reply:
thank you!!